src/components/Section/Section.jsx

```diff
@@ -70,8 +70,31 @@ class Section extends React.Component {
   }
 
   render() {
-    const sections = this.createSections()
-    return <SectionViews current={this.props.section}>{sections}</SectionViews>
+    return (
+      <Switch>
+        {/* REFACTORED - These sections are rendered via <Route>s */}
+        <Route path="/form/identification" component={Identification} />
```

Comment:
I'm struggling to wrap my head around how these routes are going to work.
The SF86 will have:

```
<Route path="/form/relationships" component={Relationships} />
```
...but the SF85 will not have that section.

How can we make sure that route is not exposed? Is there a way we can use the form type config to generate this?

Reply:
Yes, the routes can be generated from a config (or omitted from the JSX) based on the `formType` value.